Readme.md

```diff
@@ -125,9 +125,22 @@ Custom HTML file:
 ```html
 $ echo \
   '<!doctype html>
-  <link rel="stylesheet" href="/style.css" />
-  <script src="/elm.js"></script>
-  ' > index.html
+<html>
+  <head>
```

Comment:
I think it is a bit confusing to keep them out there, because people are supposed to add a title and stuff, so I would be in favor of keeping them there.